ccmgradm/main.go

```diff
@@ -77,19 +88,30 @@ func init() {
 	}
 }
 
+const (
+	ProductionBaseDir = "/usr/share/ccmgr/"
+)
+
 func reloadDaemon() error {
 	_, err := httpCli.Get(address)
 	return err
 }
 
 func main() {
-	fmt.Println("ClusterControl Manager - admin CLI v1.1")
+	fmt.Println("ClusterControl Manager - admin CLI v2.2")
 
 	arg.MustParse(&args)
 
 	// most options require a config update as well
 	saveAndReload := true
 
+	if production == "true" {
```

Comment:
it's usually better to have it the other way around - production is the default, and then "dev" mode can be optionally enabled with flags or something

Reply:
How about this approach? https://github.com/severalnines/cmon-proxy/pull/28/commits/b2759199e30a16cf99a4e0649ab4f795f592bb22#diff-7d2c6a9570d0e5f6f395317d2cbdcb1073ef07d68eb4cea44ea8d05753dde9cfR7

default is prod